app/services/text_message_verification_code_service.rb

```diff
@@ -9,19 +11,22 @@ def initialize(phone_number:, locale: :en, visitor_id:, client_id: nil, service_
 
   def request_code
     verification_code, access_token = TextMessageAccessToken.generate!(sms_phone_number: @phone_number, client_id: @client_id)
+    outgoing_message_status = OutgoingMessageStatus.find_or_create_by!(message_type: :sms, parent: access_token)
     twilio_response = TwilioService.send_text_message(
       to: @phone_number,
       body: I18n.t("verification_code_sms.with_code",
                    service_name: @service_data.service_name,
                    locale: @locale,
                    verification_code: verification_code
-      ).strip
+      ).strip,
+      status_callback: twilio_update_status_url(outgoing_message_status.id, locale: nil),
```

Comment:
I think the reason it is passed as nil is we didn't want to mount the twilio endpoints at both /en/outgoing_text_messages and /es/outgoing_text_messages etc because they don't actually contain localized content

though when I play around in the console I get the same result doing
`twilio_update_status_url(12345, locale: nil)` -> `/webhooks/twilio/update_status/12345`
and
`twilio_update_status_url(12345)` -> `/webhooks/twilio/update_status/12345`

so maybe it's a little cargo culty

it might be a different story when called from controller-land rather than a service though, so I think we left the explicit nil in there in case it gets copied from one place to another